packages/styles/src/themes/ds5Theme.ts

```diff
@@ -78,6 +78,12 @@ const ds5Theme = makeTheme((theme) => ({
     baseCheckBox: {
       hoverColor: theme.colors.acce2s,
     },
+    baseRadio: {
```

Comment:
DS3 and DS5 have the same values here. I think the hover color in [DS5](https://designsystem.hitachivantara.com/6c705d900/v/30842/p/298efd-selection/b/411129/i/119173171) is `acce2s` and has a border radius of `2px`.